src/devices/auriol_4ld5661.c

```diff
@@ -11,16 +13,18 @@
 */
 
 /**
-Lidl Auriol 4-LD5661/4-LD5972/4-LD6313 sensors.
+Lidl Auriol 4-LD5661/4-LD5972/4-LD6313 and Aldi Sempre 4-AH0423-4 sensors.
 
 See also issues #1857, #2631 and PR #2633
 
 Data layout:
 
-    II B TTT F RRRRRR
+    II B F CC TTT F RRRRRR
```

Comment:
The chars are usually nibbles. Mark just `F` then detail what's in the flags in the list.

Reply:
Redid the current documentation to reflect the unknown bits 9, 10 and 11 of the payload.